packages/core/src/components/dialog/_multistep-dialog.scss

```
@@ -110,21 +111,34 @@ $step-radius: $pt-border-radius * 2 !default;
     background-color: $pt-dark-icon-color-disabled;
   }
 
-  .#{$ns}-active & {
+  .#{$ns}-active.#{$ns}-dialog-step-viewed & {
     background-color: $blue4;
   }
+
+  .#{$ns}-dialog-step-viewed & {
+    background-color: $gray3;
+  }
 }
 
 .#{$ns}-dialog-step-title {
-  color: $gray1;
+  color: $pt-text-color-disabled;
+  flex: 1;
   padding-left: 10px;
 
   .#{$ns}-dark & {
     color: $pt-dark-text-color-disabled;
   }
 
   // by default, step title is active only when the step is selected
-  &.#{$ns}-active {
+  .#{$ns}-active.#{$ns}-active.#{$ns}-dialog-step-viewed & {
```

duplicate class name here. should be:

```suggestion
  .#{$ns}-active.#{$ns}-dialog-step-viewed & {
```